assets/armsnippets.jsonc

```diff
@@ -140,6 +140,120 @@
+            "            {",
+            "                \"name\": \"${8:appGatewayFrontendIP}\",",
+            "                \"properties\": {",
+            "                    \"PublicIPAddress\": {",
+            "                        \"id\": \"[resourceId('Microsoft.Network/publicIPAddresses', '${3:publicIP}')]\"",
+            "                    }",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"frontendPorts\": [",
+            "            {",
+            "                \"name\": \"${9:appGatewayFrontendPort}\",",
+            "                \"properties\": {",
+            "                    \"Port\": 80",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"backendAddressPools\": [",
+            "            {",
+            "                \"name\": \"${10:appGatewayBackendPool}\",",
+            "                \"properties\": {",
+            "                    \"BackendAddresses\": [",
+            "                      {",
+            "                          \"IpAddress\": \"${11:backendIpAddress1}\"",
+            "                      },",
+            "                      {",
+            "                          \"IpAddress\": \"${12:backendIpAddress2}\"",
+            "                      }",
+            "                    ]",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"backendHttpSettingsCollection\": [",
+            "             {",
+            "                \"name\": \"${13:appGatewayBackendHttpSettings}\",",
+            "                \"properties\": {",
+            "                    \"Port\": 80,",
+            "                    \"Protocol\": \"${14|Http,Https|}\",",
+            "                    \"CookieBasedAffinity\": \"Disabled\"",
+            "                }",
+            "            }",
+            "          ],",
+            "          \"httpListeners\": [",
+            "            {",
+            "                \"name\": \"${15:appGatewayHttpListener}\",",
+            "                \"properties\": {",
+            "                  \"FrontendIPConfiguration\": {",
+            "                    \"Id\": \"[resourceId('Microsoft.Network/applicationGateways/frontendIPConfigurations', '${1:applicationGatewayName}', '${8:appGatewayFrontendIP}')]\"",
+            "                  },",
+            "                  \"FrontendPort\": {",
+            "                    \"Id\": \"[resourceId('Microsoft.Network/applicationGateways/frontendPorts',  '${1:applicationGatewayName}', '${9:appGatewayFrontendPort}')]\"",
+            "                  },",
+            "                    \"Protocol\": \"${16|Http,Https|}\",",
+            "                    \"SslCertificate\": null",

```

Comment:
Looks like `SslCertificate` should be an object.

https://docs.microsoft.com/en-us/azure/templates/microsoft.network/2019-09-01/applicationgateways#applicationgatewayhttplistenerpropertiesformat-object

So, results should look like this:

```
                            "SslCertificate": {
                              "id": "null"
                            }
```

Reply:
@ManulnNZ apologies for the delay here. This on here does not seem to be resolved.